CHANGELOG.md

```diff
@@ -5,11 +5,29 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## v1.1.17 (2023-01-18)
+
+### Changed
+
+- Modified how the `Organization.is_rbac_permissions_enabled` flag is set,
+based on whether we are dealing with an open-source, or cloud installation
+- Backend implementation to support direct user/schedule paging
+- Changed documentation links to open in new window
+- Remove helm chart signing
+- Changed the user's profile modal to be wide for all tabs
 
 ### Added
 
 - Added state filter for alert_group public API endpoint.
+- Enrich user tooltip on Schedule page
+- Added redirects for old-style links
+
+### Fixed
+
+- Updated typo in Helm chart values when specifying a custom Slack command name
+- Fix for web schedules ical export to give overrides the right priority
+- Fix for topnavbar to show initial loading inside PluginPage
+- Minor fixes
```

Comment:
```suggestion
```

I would suggest we stray away from using "Minor fixes" since it's ambiguous and doesn't give user's much context